CMakeLists.txt

```diff
@@ -514,6 +514,7 @@ list(APPEND SYMBOLS_TO_CHECK
     inet_pton
     gettimeofday
     signal
+    socketpair
```

Comment:
The same should be added into configure.ac - https://github.com/libevent/libevent/blob/7f5b8f1909c9cecf26eb631ce2627a9c109dd273/configure.ac#L276

Reply:
I've done it here - https://github.com/libevent/libevent/pull/1557, authorship had been preserved.

I will close this PR.